### PR TITLE
Change links from https://access.redhat.com/documentation to equivalent in manageiq.org

### PR DESCRIPTION
**Pull request by chessbyte**
Note, there are still some links remaining for which there are currently no equivalents in manageiq.org.

```sh
$ ag "access.redhat.com/documentation/en-us/red_hat_cloudforms"
_includes/smartstate.md:3:Support](https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html-single/support_matrix/#smart_state_analysis_support)
_includes/smartstate.md:5:Analysis](https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html-single/managing_providers/#running-a-smartstate-analysis)
creating_a_service_for_vm_provisioning/_topics/provisioning_a_vm_using_ssui.md:31:Guide](https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html-single/self_service_user_interface_guide/)
managing_infrastructure_and_inventory/_topics/data_optimization_features.md:7:Machine](https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html-single/deployment_planning_guide/#planning-where-to-put-a-new-virtual-machine)
managing_infrastructure_and_inventory/_topics/container_entities.md:106:OpenSCAP*](https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html-single/scanning_container_images_in_cloudforms_with_openscap/)
```

Will address these in future PRs as links are created in manageiq.org.

### CODE REVIEW

**Comment by Fryguy** [comment]
if we are referencing docs from other docs, we may want to use relative paths instead

**Comment by chessbyte** [comment]
> if we are referencing docs from other docs, we may want to use relative paths instead

@Fryguy need to come up with some standards for this, then I can go ahead and do this.  Feels odd that we are making changes in this repo for URLs which will be accessible from https://www.manageiq.org/docs/

**Comment by Fryguy** [comment]
I thought we had discussed and found that relative urls work and so are best.

File: _includes/create-playbook-service-catalog-item.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
     inventory. Check your inventory and add the appropriate resources
     before creating an Ansible service. For more information, see
     [Automation Management
-    Providers](https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.5/html-single/managing_providers/#automation_management_providers)
```
Comment (chessbyte) [inline _includes/create-playbook-service-catalog-item.md:8]: @Fryguy So, as an example, this [link](https://www.manageiq.org/docs/reference/latest/managing_providers/index.html#automation-management-providers) would become `../managing_providers/index.html#automation-management-providers` ?

**Comment by chessbyte** [comment]
Closing since this work was done by @dehawkins512 in the following PRs:
- https://github.com/ManageIQ/manageiq-documentation/pull/1489
- https://github.com/ManageIQ/manageiq-documentation/pull/1495
- https://github.com/ManageIQ/manageiq-documentation/pull/1498
- https://github.com/ManageIQ/manageiq-documentation/pull/1507

**Pull request closed without merging**